### ISSUE TITLE
bug: Translate3d over voiceover badly calculates the selected area. Ionic5

### ISSUE THREAD

**Issue by iuliatgp** [ionic-team/ionic-framework#24992]
[x] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
[x] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
[x] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

**Ionic version:**

[] **4.x**
[x] **5.x**

**I'm submitting a ...** 

[x] bug report
[ ] feature request

**Current behavior:**
 I would like to report the following bug connected to the ion-virtual-scroll ionic element, and with it’s translate3d formula.
So, we have a list of users… Sometimes this list can contain up to 3k of elements. We decided to pass from a normal generation of the user’s list to a virtual loading one using ion-virtual-scroll.

Everything works fine in normal use app.
The issue comes when you pass on Iphone in voiceover mode. On swipe to right (navigation towards next element) and at each swipe, the voiceover highlighted area gets an exponential calculated height.

**Expected behavior:**
On swipe towards right, the highlight voiceover area is only the element's height. 

**Steps to reproduce:**
on right swipe… first element is border highlight well.
another swipe… the second element is being read but the voiceover selection contains the area of two elements. (second and third element)
another swipe… the third element is being read but the voiceover selection contains the area of three elements (starting from the third element down: an area associated to 3 - 4 - 5th element)
and so on… until the border selection area is higher than the phone screen area, and the scroll gets blocked.

In inspect, the elements have the correct height measure. Trying to work on the translate3d we notice that is the translate3d function that rules the border selection area.

**Related code:**
Here is the header definition:
```
<ion-virtual-scroll [items]=“userList” approxItemHeight=“90px” approxHeaderHeight=“30px”
[itemHeight]=“itemHeightFn” [headerFn]=“myHeaderFn” [footerFn]=“myFooterFn” [headerHeight]=“headerHeightFn”>
```
So, indeed, as it was advised from the component’s documentation, we do set a height of the item. itemHeightFn is also 90px.
transform: translate3d(0, 0, 0). Starts from point Zero.

**Other information:**
The scroll solution has been taken from here:
https://ionicframework.com/docs/api/virtual-scroll

IOS version does not matter.. tested on IOS 14.x and also 15.x

**Ionic info:** 

```
Ionic:

   Ionic CLI : 6.13.1

Utility:

   cordova-res : not installed
   native-run  : 1.3.0

System:

   NodeJS : v10.15.3
   npm    : 6.4.1
   OS     : Windows 10
```

**Comment by liamdebeasi** [comment]
Thanks for the issue! With the release of Ionic 6 we deprecated `ion-virtual-scroll` in favor of JavaScript Framework solutions. As a result, we have a freeze on new development with the exception of critical bugs, such as security issues.

Using these new virtual scrolling solutions should provide a better developer experience, more performant app, and fewer bugs! We have created migration guides for the main 3 JavaScript Frameworks Ionic supports to make this migration easy for developers.

Migration for Angular users: https://ionicframework.com/docs/angular/virtual-scroll
Migration for React users: https://ionicframework.com/docs/react/virtual-scroll
Migration for Vue users: https://ionicframework.com/docs/vue/virtual-scroll  

The documentation for `ion-virtual-scroll` still exists at https://ionicframework.com/docs/api/virtual-scroll.

`ion-virtual-scroll` will be removed in Ionic 7. Developers do **not** need to migrate off `ion-virtual-scroll` in order to upgrade to Ionic 6 or to continue using it in Ionic 4/5.

I am going to close this issue. Thank you!
